proposals/0167-app-services.md

```diff
@@ -390,6 +392,118 @@ A navigation service is defined as a service that is currently listed as the nav
++	</struct>
+```
+
+###### RPCs to be handled:
+ - OnVoiceAssistantActivated
+
+###### New RPCs for Voice Assistant
+
+```xml
++	<function name="OnVoiceAssistantActivated" functionID="OnVoiceAssistantActivatedID" messagetype="notification">
++		<description>Activate voice session for VoiceAssistant service.</description>
++		<param name="triggerSource" type="VoiceAssistantTrigger" mandatory="true">
++			<description>Informs the voice assistant of which source triggered the event.</description>
++		</param>
++		<param name="triggerInfo" type="String" mandatory="false">
++			<description>Any extra information about the trigger. This should include the wake word used and any other speech strings recognized.</description>
++		</param>
++	</function>
++
++	<enum name="VoiceAssistantOperatingMode">
++		<element name="PUSH_TO_TALK_BUTTON"/>
++		<element name="WAKE_WORD"/>
++	</enum>
+```
+
+###### Add VoiceAssistantOperationCapabilities
+
+The application that has received `OnVoiceAssistantActivated` must refer to the `VoiceAssistantOperationCapabilities` shown below. Then, the application can start a voice session with a method supported by the HMI.
+
+```xml
++	<enum name="VoiceAssistantOperatingMode">
++		<element name="AUDIO_PASSTHRU"/>
++		<element name="MICROPHONE_DIRECT"/>
++	</enum>
++
++	<struct name="VoiceAssistantOperationCapabilities">
++		<param name="voiceAssistantOperation" type="VoiceAssistantOperatingMode" minsize="1" maxsize="2" array="true" mandatory="false"/>
++	</struct>
+...
+	<interface name="VR" version="1.1.0" date="2017-04-27">
+...
+		<function name="GetCapabilities" messagetype="response">
+			<param name="vrCapabilities" type="Common.VrCapabilities" minsize="1" maxsize="100" array="true" mandatory="false">
+				<description>Types of input recognized by VR module.</description>
+			</param>
++			<param name="voiceAssistantOperationCapabilities" type="Common.VoiceAssistantOperationCapabilities" mandatory="false">
++				<description>See VoiceAssistantOperationCapabilities.</description>
++			</param>
+		</function>
+	</interface>
+```
+
+##### Ways to launch Voice Assistant
+
+Apple CarPlay and Google Android Auto are examples where NativeVR and voice assistant are mixed up. However, there are ways on how to differentiate the use (of NativeVR and voice assistant in Apple CarPlay or Google Android Auto).
+
+- When Apple CarPlay or Google Android Auto is connected, pressing the talk button launches Apple CarPlay or Google Android Auto voice assistant while others launch NativeVR.
+- A short press of talk button launches NativeVR; a long press launches Apple CarPlay or Google Android Auto voice assistant.
+
+In addition, there are also home devices equipped with voice assistant such as Amazon Echo and Google Nest. But, it is unlikely that other voice assistants will get mixed up because those devices use (their own) exclusive voice assistants.
```

Comment:
```suggestion
In addition, there are also home devices equipped with voice assistant such as Amazon Echo and Google Nest. But, it is unlikely that other voice assistants will get mixed up because those devices use their own exclusive voice assistants.
```